shared_model/builders/protobuf/proto_transaction_builder.hpp

```diff
@@ -25,38 +25,25 @@
 
 namespace shared_model {
   namespace proto {
-    template <int S = 0>
-    class TemplateTransactionBuilder {
+    class TransactionBuilder {
      private:
-      template <int>
-      friend class TemplateTransactionBuilder;
-
-      enum RequiredFields { Command, CreatorAccountId, TxCounter, TOTAL };
-
-      template <int s>
-      using NextBuilder = TemplateTransactionBuilder<S | (1 << s)>;
-
       iroha::protocol::Transaction transaction_;
 
-      template <int Sp>
-      TemplateTransactionBuilder(const TemplateTransactionBuilder<Sp> &o)
-          : transaction_(o.transaction_) {}
-
      public:
-      TemplateTransactionBuilder() = default;
+      TransactionBuilder() = default;
 
-      NextBuilder<CreatorAccountId> creatorAccountId(
+      TransactionBuilder creatorAccountId(
```

Comment:
Something should be done with the swig so it's able to handle proper builder. I don't really like to merge this fixes